### PR TITLE
docs: community guides

### PR DESCRIPTION
**Pull request by aviv926**
Following on from PR #8808, this PR adds a Community Guides section.

@bo0tzz Can we add a new Guides section to the discussions or a new label for more convenient sorting?

### CODE REVIEW

**Comment by bo0tzz** [comment]
I've added a new guides section to the discussion.

File: docs/docs/community-guides.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Community Guides
+
+This page lists community guides that are write around Immich, but not officially supported by the development team.
```
Comment (bo0tzz) [inline docs/docs/community-guides.mdx:3]:
```suggestion
This page lists community guides that are written around Immich, but not officially supported by the development team.
```

**Review by bo0tzz — APPROVED** [review]

**Comment by aviv926** [comment]
> I've added a new guides section to the discussions.

Thanks, I'll move what's relevant there

File: docs/src/components/community-guides.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        <p className="m-0 text-sm text-gray-600 dark:text-gray-300">
+          <a href={url}>{url}</a>
+        </p>
+      </div>
+      <div className="flex">
+        <Link
+          className="px-4 py-2 bg-immich-primary/10 dark:bg-gray-300  rounded-full hover:no-underline text-immich-primary dark:text-immich-dark-bg font-bold uppercase"
+          to={url}
+        >
+          View Guides
```
Comment (jrasm91) [inline docs/src/components/community-guides.tsx:51]:
```suggestion
          View Guide
```

**Review by jrasm91 — APPROVED** [review]
Nice, this seems like a good place to link to relevant information without cluttering up the website.

**Pull request merged**